### ISSUE TITLE
"Advanced Grouping" with "albumartist" forces display of album art

### ISSUE THREAD

**Issue by Clementine-Issue-Importer** [clementine-player/Clementine#2631]
_From [scientif...@gmail.com](https://code.google.com/u/113584112684286734356/) on January 19, 2012 02:43:02_

What steps will reproduce the problem? 1. Configure the library's display: "Group by" > "Advanced Grouping"
1. Select any combination of sorting options, provided the first level is "albumartist." For sake of illustration I'll use "album" for the second level, and leave the third level empty. What is the expected output? What do you see instead? I would expect (or hope) to see the library appear exactly as it does under the standard "Album/Artist" sort option, only with the "artist" metadata replaced by "albumartist" metadata. But in addition, the small person icons are replaced by the cover art of the first album by that artist in the library. What version of the product are you using? On what operating system? Clementine 1.0, Windows 7 SP1 Please provide any additional information below. This additional artwork not only makes the system slow to render the covers, it also is aesthetically unappealing (in my opinion). It takes up significantly more room on the screen, meaning much more scrolling is required to locate the music I want to play. Also, the cover art for each album is displayed as soon as I open the tree to the second level, making the album cover displayed in the first level redundant.

In short, I'd appreciate an option to turn off this redundant cover art display when I use the "Advanced Grouping" feature with the "albumartist" tag.

_Original issue: http://code.google.com/p/clementine-player/issues/detail?id=2631_
